docs/DeveloperGuide.md

```diff
@@ -629,6 +692,35 @@ testers are expected to do more _exploratory_ testing.
    1. Test case: `deleteorder x o/1` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+### Adding a contact as favourite
```

Comment:
No mention of the warning in the event some are already favourites, which I think is alright since the functionality you listed is all still accurate. Mentioning this in case you missed them out, but no change required imo